src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -418,6 +416,53 @@ class WorkspaceService(dataSource: DataSource, workspaceDAO: WorkspaceDAO, entit
       // to ensure it is evaluated within the transaction
       RequestComplete(methodConfigurationDAO.list(workspaceNamespace, workspaceName, txn).toList)
     }
+
+  // TODO: replace this stub with the actual expression parser
+  private def resolveInputs( methodConfig: MethodConfiguration, entity: Entity ): String =
+    """{"hello": "world"}"""
+
+  def submitJob(jobDesc: JobDescription, authCookie: HttpCookie): PerRequestMessage =
+    dataSource inTransaction { txn =>
+      val wsNamespace = jobDesc.methodName.workspaceName.namespace
+      val wsName = jobDesc.methodName.workspaceName.name
+      val mcNamespace = jobDesc.methodName.namespace
+      val mcName = jobDesc.methodName.name
+      methodConfigurationDAO.get(wsNamespace, wsName, mcNamespace, mcName, txn) match {
+        case None =>
+          RequestComplete(StatusCodes.NotFound,
+            "No method configuration named "+mcNamespace+"."+mcName+" in workspace "+wsNamespace+"."+wsName)
+        case Some(methodConfig) =>
+          val entType = methodConfig.rootEntityType
+          val entName = jobDesc.entityName
+          entityDAO.get(wsNamespace, wsName, entType, entName, txn) match {
+            case None =>
+              RequestComplete(StatusCodes.NotFound,
+                "No entity of type "+entType+" named "+entName+" in workspace "+wsNamespace+"."+wsName)
+            case Some(entity) =>
+              Try { resolveInputs(methodConfig,entity) } match {
+                case Failure(error) =>
+                  RequestComplete(StatusCodes.Conflict, "Expression evaluation failed: "+error.getMessage())
+                case Success(inputs) =>
+                  val mrNamespace = methodConfig.methodNamespace
+                  val mrName = methodConfig.methodName
+                  val mrVersion = methodConfig.methodVersion
+                  methodRepoDAO.getMethod(mrNamespace, mrName, mrVersion, authCookie) match {
+                    case None => RequestComplete(StatusCodes.NotFound)
+                    case Some(agoraEntity) =>
+                      if ( agoraEntity.payload.isEmpty )
+                        RequestComplete(StatusCodes.NotFound, "Can't get method's WDL from Method Repo: ")
```

Comment:
Was there supposed to be something after the colon in this error message, e.g. the method name?